help/configure-email-visibility.md

```diff
@@ -0,0 +1,33 @@
+# Configure email visibility
+
+With privacy in mind, Zulip lets you control which
+[roles](/help/roles-and-permissions) in the organization can view your email
+address. Administrators can set the default email visibility configuration
+for new users in the organization.
+
+## Configure who can access your email address
+
+{start_tabs}
+
+{settings_tab|account-and-privacy}
+
+1. Under **Privacy**, configure **Who can access your email address**.
+
+{end_tabs}
+
+## Configure default email visibility for new users
+
+{!admin-only.md!}
+
+{start_tabs}
+
+{settings_tab|default-user-settings}
+
+1. Under **Privacy settings**, configure **Who can access user's email address**.
+
+{end_tabs}
+
+## Related articles
+
+* [Moderating open organizations](/help/moderating-open-organizations)
+* [Restrict name and email changes](/help/restrict-name-and-email-changes)
```

Comment:
Small nit: If you'd doing the URL change in the second commit, it'd be cleaner to have the first commit just rewrite the content (including everything but the URL), and the second commit just be the URL change -- here you have the file rename (and thus part of the URL change) in the first commit; this means two things:
* The first commit is broken without the second
* Git tools won't as readily display the diff for anything you didn't change in the original page.

If it's a total rewrite like this, that second item isn't a concern, and one option for avoiding the first issue is to have the first commit just contain the `url_redirects.py` configuration -- so all the links still work (albeit with an extra redirect) after just the first commit.

Reply:
Done! I usually try to avoid these issues, but I guess it slipped my mind this time since this PR was a continuation from another PR. I'll keep it in mind for next time, thanks!